src/plugins/crypto/gpg.c

```diff
@@ -25,6 +25,37 @@ static inline void closePipe (int * pipe)
 }
 
 /**
+ * @brief checks whether or not a given file exists and is executable.
+ * @param file holds the path to the file that should be checked
+ * @param errorKey holds an error description if the file does not exist or if it is not executable. Ignored if set to NULL.
+ * @retval 1 if the file exists and is executable
```

Comment:
I can see how `0` indicating success can be confusing. I changed it to `1` to be consistent with the rest of the module.